manifests/pcap-api.yml

```diff
@@ -60,14 +79,31 @@ instance_groups:
               - name: pcap-api-public-endpoint
                 registration_interval: 20s
                 port: 8080
+                #                tls_port: 8080
```

Comment:
Remove if it is not needed
If port and tls_port are provided, then route_registrar takes the value of tls_port.

Reply:
See https://github.com/cloudfoundry/pcap-release/pull/23/commits/f4553fbb7302ad917ff8e4b8413eb93c7c830fe6